src/styles/card.scss

```
@@ -0,0 +1,59 @@
+.card {
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  padding-inline: 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  width: 166px;
```

and add `box-sizing: border-box` to `*` selector
```suggestion
  width: 200px;
```